### PR TITLE
Remove innerHTML assignments, non-API docs

### PR DESCRIPTION
**Pull request by Josh-Cena**
Part 1 of https://github.com/mdn/content/issues/13496. Use `innerText` where line breaks are needed, `textContent` otherwise, and `appendChild` if necessary.

### CODE REVIEW

File: files/en-us/learn/forms/how_to_build_custom_form_controls/example_4/index.md
Comment (bsmth) [inline files/en-us/learn/forms/how_to_build_custom_form_controls/example_4/index.md:1]: Luckily this is not referenced in an external repo :)

File: files/en-us/learn/javascript/building_blocks/conditionals/index.md
Comment (bsmth) [inline files/en-us/learn/javascript/building_blocks/conditionals/index.md:1]: Needs an update here: https://github.com/mdn/learning-area/blob/main/javascript/building-blocks/calendar-finished.html#L70

File: files/en-us/web/css/@container/index.md [hunk 1]
```diff
@@ -154,11 +154,11 @@ You can then use the `@container` at-rule to apply styles to the element with th
 ```js hidden
 const post = document.querySelector(".post");
 const span = document.createElement("span");
-span.innerHTML = ".post width: " + post.clientWidth + "px";
+span.textContent = ".post width: " + post.clientWidth + "px";
```
Comment (bsmth) [inline files/en-us/web/css/@container/index.md:157]: Oops, I think that was my fault :)

**Review by bsmth — APPROVED** [review]
LGTM, thank you! We might want to get the learning-area ones in building-blocks as a follow-up

**Comment by Josh-Cena** [comment]
Yeah, I think we have several external examples that need updates here.

**Pull request merged**